### DISCUSSION TITLE
nnunet implementation for multiple labels

### DISCUSSION THREAD

**Discussion by bereket-meley** [Project-MONAI/tutorials#1507] [Q&A]
Where to specify the dataset has multiple labels?
On nnunet we used to specify on dataset.json as  "modality": {
    "0": "MRI"
  },
  "labels": {
    "0": "background",
    "1": "liver",
    "2": "PV",
    "3": "IVC",
    "4": "HV"
  },

**Comment by KumoLiu** [comment]
Hi @bereket-meley, I think you can prepare something like this:
https://github.com/Project-MONAI/tutorials/blob/main/auto3dseg/tasks/hecktor22/input.yaml

Hope it helps, thanks!
